index.bs

```diff
@@ -584,16 +584,14 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. For each [=authenticator=] currently available on this platform, process each option in 
     |options|.{{MakeCredentialOptions/authenticatorSelection}} by checking if the [=authenticator=] satisfies the 
     requirement of the option. 
-        1. Check if |attachment| is [=present|not present=] or its value matches
-            |authenticator|'s attachment modality, [=continue=].
-        1. If |requireResidentKey| is set to |true|, check if the |authenticator| is capable of storing 
-            [=Client-Side-Resident Credential Private Key=]. If so, [=continue=]. If not, return a {{DOMException}} whose name
-            is "{{QuotaExceededError}}".
-        1. If the [=authenticator=] is capable of all criterias as listed above, [=set/append=] [=authenticator=] to 
-            |currentlyAvailableAuthenticators|. 
-        1. If there is no [=authenticator=] that satisfies the criterias listed in 
-            |options|.{{MakeCredentialOptions/authenticatorSelection}}, return a {{DOMException}} whose name is 
-            "{{ConstraintError}}", and terminate this algoritm.
+        1. If |attachment| is [=present|not present=] or its value matches
+            |authenticator|'s attachment modality, then [=iteration/continue=]. Otheriwse, [=iteration/break=].
```

Comment:
Both continue and break will skip the remaining substeps in this sequence. They act like the same keywords in C, Java, Python, etc.